### ISSUE TITLE
Порезать доступ офицерам

### ISSUE THREAD

**Issue by Spaint** [animusdev/blue#641]
Медбей, карго, куча других отделов. Не дохуя ли? Даже только для старших. В идеале таким доступом должен владеть только комиссар. Хотят доступ к сенсорам и консоли заказов - пускай просят у ХоПа.

**Comment by RandomColonist** [comment]
Да, давайте как на бее - выбивать двери по тревоге, если нет хоса или хопа. К тому же - такой доступ только у вардена (который сидит в бриге всю смену) и двух старших офицеров, которые априори не должны быть овощами кончеными (если заметите обратное - никто не мешает выдать джоббан или ингейм подать жалобу).

**Comment by Spaint** [comment]
В том и суть, что они должны не спокойно разгуливать по любой части станции, а сосать и ломать двери, если что-то происходит где-то кроме брига. Ну, или звать кого-то/получать доступы. Не в первый раз вижу, как щиткуриты хозяйничают в карго/медбее при живом и активном персонале.

**Comment by RandomColonist** [comment]
>должны не спокойно разгуливать по любой части станции, а сосать и ломать двери, если что-то происходит где-то кроме брига.

Я вот не понимаю прикола в беспомощном сб, да и кто сказал, что они ДОЛЖНЫ сосать и ломать двери?

**Comment by Spaint** [comment]
Я вот не понимаю прикола во всесильном СБ. То, что у сотрудников одного отдела нет доступа в другом - хорошо и сбалансированно, менять это смысла нет. Почему ассистенты, медики, даже инженеры должны просить доступ или пробивать себе путь, а офицеры - нет? Можно провести аналогию с теми же медиками и раненными в закрытых отсеках.

**Comment by FinnTheTurtle** [comment]
С одной стороны, это помогает действовать оперативно, а не "ОТКРОЙТЕ ПОЖАЛУЙСТА У ВАС ТАМ УБИЙСТВО". С другой же, этим, как и обычно, злоупотребляют. Явно стоит что-то сделать.

**Comment by Divix01** [comment]
Сделать возможность устанавливать картам доступ в зависимости от уровня тревоги?

- Зеленый - в бриг и техи;
- Синий - частичный доступ в медбей, карго и т.д. (но не в хирургию, не в ядро с материей и т.п.);
- Красный - во все отсеки, за исключением оружейки, хранилища и каюты кэпа/хоса.

**Comment by alfacom** [comment]
@Divix01 Напомни пожалуйста, кто уровень тревоги меняет?

**Comment by Divix01** [comment]
@alfacom у меня для тебя аж два ответа на твой вопрос!
1) Никто. Это будет поводом поднимать уровень тревоги при ЧП;
2) СБ. Наконец-то будет больше бугурта у персонала за уровень тревоги и будет больше кары за "Ну я типо по РП поднимаю тревогу, я так отыгрываю!".
